opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -103,6 +103,16 @@ message Span {
   // field must be empty. The ID is an 8-byte array.
   bytes parent_span_id = 4;
 
+  // Flags, a bit field. 8 least significant bits are the trace flags as
+  // defined in W3C Trace Context specification. 24 most significant bits are reserved
+  // and must be set to 0. Readers must not assume that 24 most significant bits
+  // will be zero.  When writing new spans, SDKs must set the most-significant 24-bits
```

Comment:
Nit: technically Collector can create spans too, not only SDKs. So perhaps simply drop the reference to SDKs.

Reply:
I think what the bits should be set to in new objects should not be described in the protocol.